File: python/PROBLEMS/1036.py
e = input().split(' ')

a =float(e[0])
b =float(e[1])
c =float(e[2])

delta = pow(b,2)-(4 * a * c)

if delta < 0 or a <=0:
    print("Impossivel calcular")
else:
    print("R1 = {:.5f}".format((-b + pow(delta, 0.5))/(2*a)))
    print("R2 = {:.5f}".format((-b - pow(delta, 0.5))/(2*a)))

File: c/PROBLEMS/1036.c
#include <stdio.h>
#include <math.h>
int main(){
    double a, b, c;

    scanf("%lf %lf %lf", &a, &b, &c);

    double delta = pow(b,2)-(4 * a * c);

    if ((delta < 0) || (a <=0)){
        printf("Impossivel calcular\n");
    }
    else{
        printf("R1 = %.5f\n", ((-b + pow(delta, 0.5))/(2*a)));
        printf("R2 = %.5f\n", ((-b - pow(delta, 0.5))/(2*a)));
    }
}